connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/webhook/WebhookConnectorRegistry.java

```diff
@@ -54,6 +74,18 @@ public void register(ActiveInboundConnector connector) {
     }
   }
 
+  private static void logIfWebhookPathDeprecated(ActiveInboundConnector connector, String webhook) {
+
+    if (!CURRENT_WEBHOOK_PATH_PATTERN.matcher(webhook).matches()) {
+      String message =
+          DEPRECATED_WEBHOOK_MESSAGE_PREFIX + webhook + DEPRECATED_WEBHOOK_MESSAGE_SUFFIX;
+      Activity activity =
+          Activity.level(Severity.WARNING).tag(WEBHOOK_CONNECTOR_REGISTRY).message(message);
```

Comment:
As this string will be shown in the webmodeler as a tag I am not sure how useful the information `WebhookConnectorRegistry` would be for a user. I would go with `Warning` or `Validation` or something. Maybe @YanaSegal has a good idea. 

![Bildschirmfoto 2024-01-25 um 09 44 54](https://github.com/camunda/connectors/assets/1607241/23c89fa8-1cfa-4c64-8645-e23a3211c506)

Reply:
Good idea!

Given this context, even "Warning" would be better than "WebhookConnectorRegistry", so I changed the tag to "Warning" for now. Will update based on Yana's feedback :)